toptal/picasso
# feat: update skeleton loader colors

[FX-3824](https://toptal-core.atlassian.net/browse/FX-3824)

### Description

Updated the color gradient for Skeleton loaders so it is usable on all backgrounds (well, apart from one specific grey one)

### How to test

- Take a look at the visual tests
- Take a look at the deployed Storybook, go to `SkeletonLoader` and scroll down to the [latest story](https://picasso.toptal.net/fx-3824-skeleton-loaders-loaders-on-dark-backgrounds/?path=/story/components-skeletonloader--skeletonloader#different-background-color) added for dark backgrounds
- See if the new shared colors for loaders make sense (the location in the code and naming) 

### Screenshots

| Before.                                 | After.                                  |
| --------------------------------------- | --------------------------------------- |
| Insert screenshots or screen recordings | Insert screenshots or screen recordings |

### Development checks



> All **_development checks_** should be done and set checked to pass the
> **GitHub Bot: TODOLess** action

<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run package:alpha-release` - Release alpha version
- `@toptal-anvil ping reviewers` - Ping FX team for review

</details>

<details>
<summary>PR Review Guidelines</summary>
<br />

#### When to approve? ✅

**You are OK** with merging this PR and

1. You have no extra requests.
2. You have optional requests.
   1. Add `nit:` to your comment. (ex. `nit: I'd rename this variable from makeCircle to getCircle`)

#### When to request changes? ❌

**You are not OK** with merging this PR because

1. Something is broken after the changes.
2. Acceptance criteria is not reached.
3. Code is dirty.

#### When to comment (neither ✅ nor ❌)

**You want your comments to be addressed** before merging this PR in cases like:

1. There are leftovers like unnecessary logs, comments, etc.
2. You have an opinionated comment regarding the code that requires a discussion.
3. You have questions.

#### How to handle the comments?

1. An owner of a comment is the only one who can resolve it.
2. An owner of a comment must resolve it when it's addressed.
3. A PR owner must reply with ✅ when a comment is addressed.

</details>


[FX-3824]: https://toptal-core.atlassian.net/browse/FX-3824?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
